src/parameters.rs

```diff
@@ -24,21 +24,23 @@ impl_serialization_traits!(ThresholdParameters<CipherSuite>);
 impl<C: CipherSuite> ThresholdParameters<C> {
     /// Initialize a new set of threshold parameters.
     ///
-    /// Will panic if one of the following condition is met:
+    /// In debug mode, will panic if one of the following condition is met:
     ///  - n equals 0
     ///  - t equals 0
-    ///  - n < t
+    ///  - n <= t
```

Comment:
A bit unsure about what the expected behaviour is: can the number of participants be equal to the threshold or not? The docs said "no" but the assert in code allowed it, so I updated the docs here.

Reply:
I think we can allow t=n (the case is mentioned in the ICE-FROST paper), but the signature will fail because there won't be enough uncorrupted shares I believe.

Reply:
Why would the signature fail? Having all participants required is fine to generate a valid signature (as long as they are ALL behaving honestly). This is even tested, the doc just has a typo which is fixed here by David.

Reply:
Ah yes nevermind, sorry, I don't know why, but I somehow thought of the number of failures instead of the threshold (which doesn't really make sense)... It's Monday I guess...